src/opengl.c

```diff
@@ -222,7 +222,7 @@ static void OpenGLRenderer_EndDraw() {
     if (!g_opengl_es)
       glTexImage2D(GL_TEXTURE_2D, 0, GL_RGBA, g_draw_width, g_draw_height, 0, GL_BGRA, GL_UNSIGNED_INT_8_8_8_8_REV, g_screen_buffer);
     else
-      glTexImage2D(GL_TEXTURE_2D, 0, GL_RGBA, g_draw_width, g_draw_height, 0, GL_BGRA, GL_UNSIGNED_BYTE, g_screen_buffer);
+      glTexImage2D(GL_TEXTURE_2D, 0, GL_BGRA, g_draw_width, g_draw_height, 0, GL_BGRA, GL_UNSIGNED_BYTE, g_screen_buffer);

```

Comment:
GL_BGRA doesn't appear to be a valid value for this parameter?

Reply:
I was getting a GL_INVALID_VALUE (0x501) error until I changed the internal format to match the format. Having them both as GL_RGBA also worked, but I got a black screen with GL_RGBA as the internal format and GL_BGRA as the format. I found the solution on this page: https://stackoverflow.com/questions/50717395/how-to-load-a-bgra-image-texture-in-gles2

It may be an issue specific to android, but it fixed the errors on both an x86-64 android emulator and physical android device with an Adreno 630 GPU.